pubspec.yaml

```diff
@@ -50,6 +50,8 @@ dependencies:
   # Data visualisation
   syncfusion_flutter_charts: ^20.2.48
 
+  # Word cloud
+  flutter_scatter: ^0.2.0
```

Comment:
Can you also add the `pubspec.lock` file, when you add dependencies to the project it's okay to include it too

Reply:
Addressed in commit 6483a786a2ac44e1e265ed3333bbabb9fd9c4b20